packages/app-desktop/app.reducer.ts

```diff
@@ -308,6 +309,10 @@ export default function(state: AppState, action: any) {
 			};
 			break;
 
+		case 'PLUGIN_SEARCH_QUERY':
+			newState = Object.assign({}, state);
+			newState.pluginSearchQuery = action.pluginId;
+			break;
```

Comment:
It's possible to pass additional properties to the NAV_GO action. I think you should do that because a separate action is not needed here.

Reply:
ok, I will pass  pluginId in the props of NAV_GO action
```js
store.dispatch({
	type: 'NAV_GO',
	routeName: 'Config',
	props: {
		defaultSection: 'plugins',
		pluginId,
	},
});
```